internal/xds/translator/testdata/out/ratelimit-config/global-shared-distinct-match.yaml

```diff
@@ -1,8 +1,8 @@
-name: test-policy-1-test-namespace
-domain: test-policy-1-test-namespace
+name: test-namespace/test-policy-1
+domain: test-namespace/test-policy-1
```

Comment:
This doesn't match the domain in the ratelimit filter:

`domain: test-policy-1/test-namespace`